console/billing/how-to/redeem-voucher-code.mdx

```diff
@@ -0,0 +1,44 @@
+---
+meta:
+  title: How to redeem a Scaleway voucher 
+  description: This page explains how to redeem a Scaleway voucher 
+content:
+  h1: How to redeem a Scaleway voucher code
+  paragraph: This page explains how to redeem a Scaleway voucher 
+tags: discount voucher billing redeem 
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+A Scaleway voucher provides you with a discount or credit to use on our services. You can receive it as part of a promotional campaign or from your account manager. To redeem a voucher, you need to [create a Scaleway account](/console/account/how-to/create-an-account/).
+
+Voucher codes are unique and valid only for a defined period of time.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/billing/optimization) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```

Comment:
Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.